node/src/services/blockchain/mod.rs

```diff
@@ -1,18 +1,32 @@
+pub mod commands;
 pub mod events;
 pub mod handler;
+pub mod types;
 
+use std::sync::Arc;
+
+use sc_service::RpcHandlers;
+use sp_keystore::KeystorePtr;
+use sp_runtime::KeyTypeId;
 use storage_hub_infra::actor::{ActorHandle, ActorSpawner, TaskSpawner};
 
+use crate::service::ParachainClient;
+
 use self::handler::BlockchainService;
 
+pub const KEY_TYPE: KeyTypeId = KeyTypeId(*b"bcsv");
```

Comment:
What about `bcsvc`? Normally “service" is represented as `svc`. If you hadn’t told me before, I wouldn’t have known what `sv` is.

Reply:
Good suggestion, but `KeyTypeId`s are always 4 chars in length.

Reply:
Ah I see, we can change it later if we come up with something better.